Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi8/python-36:1-89.1589298690
```

Comment:
Please use one of the Thoth builder image

Reply:
Well, actually we can have this build by a tekton pipeline ;)

@harshad16

Reply:
This would be built by the aicoe-ci(tekton pipeline).